source/d3d11/state_tracking.hpp

```diff
@@ -37,6 +37,7 @@ namespace reshade::d3d11
 #if RESHADE_DEPTH
 		void on_set_render_targets();
 		void on_clear_depthstencil(UINT clear_flags, ID3D11DepthStencilView *dsv, bool rect_draw_call = false);
+		std::vector<std::pair<ID3D11Texture2D*, depthstencil_info>> sorted_counters_per_used_depthstencil();
```

Comment:
This should be part of `state_tracking_context` instead of `state_tracking`, since we don't want a separate instance in every device context and command list, but rather only one. Same thing applies to the `_shown_count_per_depthstencil_address` field (and to D3D12 too). Don't need to bother with merging then either.

Reply:
Done